src/community/roadmap/index.md

```diff
@@ -13,11 +13,13 @@ Some things on the roadmap might change – the purpose is to tell you what’s
 
 See our [GitHub team board](https://github.com/orgs/alphagov/projects/53) for more details on our plans and day-to-day activities.
 
-Last updated 24 June 2025.
+Last updated 17 July 2025.
 
 ## Recently shipped
 
-We’ve released [GOV.UK Frontend v5.11.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.0), which includes improvements to the Service navigation component, making it easier to use on mobile devices and offering a new inverse colour option. We’ve also added deprecation warnings for code built with the LibSass library.
+We've release [GOV.UK Frontend v5.11.1](https://github.com/alphagov/govuk-frontend/releases/tag/v5.11.0), which fixes link styles in the Service navigation on inverted backgrounds and makes govuk-shade and govuk-tint more reliable.
```

Comment:
The URL here needs correcting.

Reply:
Whoops, missed that in the copy-paste, thanks for spotting it.